### DISCUSSION TITLE
Playtime: User default template saving and loading at start

### DISCUSSION THREAD

**Discussion by joungle123** [helgoboss/helgobox#1246] [Ideas]
When you start Playtime 2 it would be great to be able to load a default template (e.g. I want to always load as MIDI) and not to select every time, it would just save a creative headache of every time selecting that option

**Comment by helgoboss** [comment]
I think this is covered by REAPER project templates (or even REAPER default projects) or REAPER track templates (can contain multiple tracks I think) already? Right now, I don't see a good reason to duplicate that within Playtime.

**Reply by joungle123** [reply]
@helgoboss okay - let me rephrase that - would it be possible to add checkbox to check 'load as default' and when you click MIDI or Audio which is basically the same in reaper (no difference in audio vs midi track) so I am not sure why we even need this screen (maybe I am wrong, in that case I may apologize) it is a bit distracting and blank canvas would be preferred or at least loading those 8 tracks as default when you click that 'checkbox' that you want it to load as default so that means no more seeing that selection.
![image](https://github.com/user-attachments/assets/584e8873-a3d9-48e0-8717-09afd16b7396)

**Reply by helgoboss** [reply]
You don't need this screen. It's just a convenience feature, completely optional. You can also just add a column manually and it will disappear.

I mentioned REAPER project/track templates because they already do what you want.

## Project templates

Do this once:

1. Start a new project and press the Playtime button
2. Configure Playtime as you wish
3. Press File => Project templates => Save project as template...
4. Choose a name for the template (e.g. "My Playtime project")

Do this whenever you want to use your pre-configured Playtime in a project:

1. Press File => Project templates => My Playtime project
2. Press the Playtime button
3. Et voila ... you will see your pre-configured Playtime

No need to add another feature.

## Track templates

If you want to add your pre-configured Playtime to an already existing project, you can do that by using track templates.

Do this once:

1. Start a new project and press the Playtime button
2. Configure Playtime as you wish
3. Select the Playtime track and all column tracks
4. Right-click the selected tracks => Save tracks as track template...
5. Choose a name for the template (e.g. "My Playtime tracks")

Do this whenever you want to add your pre-configured Playtime to an existing project:

1. Right-click somewhere below your existing tracks => Insert track from templates => My Playtime tracks
2. Press the Playtime button
3. Et voila ... you will see your pre-configured Playtime

No need to add another feature.

**Reply by joungle123** [reply]
That is an option of course, thanks. It would be nice if we could just double click to create new track. The more easy it is to manipulate a program the more creative and effortless and enjoyable music creation is which is highly important if you want to not to lose the inspiration and make great music

**Reply by helgoboss** [reply]
FYI, I'm streamlining above mentioned "Track template" approach in #1348.
